README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+# Magic Books Inc
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This application is built with real-world business specifications. It an app for a Germany fictional company *Magic Books Inc*, the application helps organize and manage the company's bookstore. It is a fast, dynamic app built with ReactJs and a few more technologies listed below.
 
-## Available Scripts
+## Technologies used:
+> technologies used in building this project include but are not limited to: 
+- EcmaScript 6
+- webpack
+- big.js
+- reactJS
+- buildpack
 
-In the project directory, you can run:
+## live Demo
+[MagicBooks.Inc](https://magicbooks-inc.herokuapp.com/)
 
-### `yarn start`
+## Project Design
+![BookStore](./demo/projectdemo.gif)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Project SetUp
+You can clone the project on your local machine by running:
+- `git clone https://github.com/ClaytonSiby/Magic-Books-Inc.git`
+then:
+-  `cd The_Calculator`
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+### install all dependencies by running:
+- `npm install`
 
-### `yarn test`
+- you can view the project live by runnig `yarn start`
+- for production release/deployment run `npm run build` to bundle a minified version of the code.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Author 
 
-### `yarn build`
+👤 **Clayton Siby**
+​
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- Github: [@ClaytonSiby](https://github.com/ClaytonSiby)
+   
+- Linkedin: [Linkedin Profile](https://www.linkedin.com/in/clayton-siby-48a8a0183/)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Twitter: [@Twitter](https://twitter.com/ClaytonSiby)
```

Comment:
## Improve social links 
```markdown
- Linkedin: [Clayton Siby](https://www.linkedin.com/in/clayton-siby-48a8a0183/)

- Twitter: [@ClaytonSiby](https://twitter.com/ClaytonSiby)

```

Reply:
Hi @christianotieno, thanks for the feedback. I still don't understand when you say `Improve social links`, what is it that I need to change here?

Reply:
What I shared was a suggestion of replacing the social placeholders that are currently present in the social links section of the README. it would be great if you implemented this suggestion by replacing the current placeholder with your actual username. For instance `@ClaytonSiby` instead of `@Twiter`. I hope this helps with the clarity.